src/calendar-app/calendar/gui/CalendarGuiUtils.ts

```diff
@@ -782,6 +782,15 @@ export function getEventType(
 		return EventType.SHARED_RO
 	}
 
+	/**
+	 * userIsOwner was introduced to keep the current behaviour(v235.240718.0)
```

Comment:
I would not mention this ( I feel like it doesn't need to be explained /here/ why userIsOwner was introduced, maybe mention it in the commit message ), just document the behavior (which you have done, thank you for that)